examples/images_contours_and_fields/contourf_demo.py

```diff
@@ -33,6 +33,7 @@
 interior = np.sqrt((X**2) + (Y**2)) < 0.5
 Z[interior] = np.ma.masked
 
+fig, ax = plt.subplots(constrained_layout=True)
```

Comment:
This creates an unused figure, which is probably not desired: 

![image](https://user-images.githubusercontent.com/23121882/42345555-f81faed4-809f-11e8-97cb-12a5a2cb2b3b.png)